contracts/dependencies/contracts.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity 0.5.0;
+
+// hack for truffle, so we have artifact for dependencies contracts for migrations and tests
```

Comment:
This file should be under `test/contracts`

Reply:
unfortunately truffle will not see it when you put it anywhere but inside `contracts/`. Under `test/` we can have only unit tests in solidity, but not contracts.